cmd/influxd/main.go

```diff
@@ -72,9 +72,15 @@ func main() {
 	// Extract name from args.
 	switch cmd {
 	case "run":
```

Comment:
Ah, OK. I see. Still seems a pity to replicate that code right there. How about:

```
case "", "run":
   if cmd == "run" {
       args = args[1:]
   }
   cmd := NewRunCommand()
    if err := cmd.Run(args...); err != nil {
        log.Fatalf("run: %s", err)
    }
}
```
Might not be worth it.